src/community/web-resources/pom.xml

```diff
@@ -0,0 +1,67 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.geoserver</groupId>
+    <artifactId>community</artifactId>
+    <version>2.9-SNAPSHOT</version>
```

Comment:
@NielsCharlier the parent version should now be ```2.10-SNAPSHOT``` as the ```2.9.x``` branch was created when ```2.9-RC1``` was released, while you were waiting for me to review this issue. Apologies for the inconvenience.